packages/eslint-plugin/src/rules/naming-convention.ts

```diff
@@ -404,6 +457,13 @@ export default util.createRule<Options, MessageIds>({
         handleMember(validators.accessor, node, modifiers);
       },
 
+      'TSAbstractMethodDefinition[computed = false]:matches([kind = "get"], [kind = "set"])'(
```

Comment:
dont know if this wasnt added on purpose previously?

Reply:
This might be a bugfix outside of the scope of this PR. Could you file a separate issue please?

Reply:
removed